### DISCUSSION TITLE
Help creating project : taas.trubudget.net

### DISCUSSION THREAD

**Discussion by dominique-AR** [openkfw/TruBudget#1672] [Q&A]
Hello everyone,
I'm new to the trubudget community, to immerse myself in the use of the application. Before installing the local version, I started using the saas version: taas.trubudget.net. I've managed to configure the profile and access the sandbox. I've created users, but I can't seem to create a project. Thanks.
![Capture d’écran du 2024-02-28 09-33-01](https://github.com/openkfw/TruBudget/assets/108085848/b6f77b36-06d2-4508-8f61-39df7641f7d6)

**Comment by georgimld** [comment]
Hello @dominique-AR, the root user cannot create projects. Please create a new user in the `/users` tab, log in via that user and then create projects.

**Reply by dominique-AR** [reply]
@georgimld . thank you very much.
